Assets/Example/Scripts/CountlyEntryPoint.cs

```diff
@@ -221,6 +221,21 @@ public async void EventWithSumAndSegmentation()
        
     }
 
+    public async void EventWithInvalidSegmentation()
+    {
+        long currentMillis = System.DateTime.UtcNow.Millisecond; // invalid data type
+        CustomDataType customDataType = new CustomDataType("foo", 49); // invalid data type
+        SegmentModel segment = new SegmentModel(new Dictionary<string, object>
+        {
+            { "Current Millis", currentMillis},
```

Comment:
To make this tests useful, add all accepted data types and a couple of types that should be rejected (2 is fine).